### PR TITLE
Install SmartOS without USB flash drive

### PR DESCRIPTION
**Pull request by optaris**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Stefan Eestermans stefan@optaris.be

### CODE REVIEW

File: tutorials/how-to-install-SmartOS/how_to_install_SmartOS_without_the_need_for_USB_flash_drive.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
+# How to install SmartOS without the need for USB flash drive?
```
[the remark this answers is not in the document]
Comment (optaris) [reply tutorials/how-to-install-SmartOS/how_to_install_SmartOS_without_the_need_for_USB_flash_drive.md]: headers were added

File: tutorials/how-to-install-SmartOS/how_to_install_SmartOS_without_the_need_for_USB_flash_drive.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+SmartOS (build: 20210701T204427Z)
+```
+
+## Conclusion
+
+This procedure explains the steps necessary to install SmartOS on the ZFS zpool on the local hard drives. The installation of the Linux Rescue system was only an intermediate step in order to boot **SmartOS** for the first time, thanks to an entry that we added in he Linux grub configuration.
+
+If all went well we now have a system that automatically boots **SmartOS** without the need for any external media.
+
+Future upgrades are as simple as installing a new image with the `piadm install` command followed by the `piadm activate` command, the new OS image will become active upon reboot.
```
[the remark this answers is not in the document]
Comment (optaris) [reply tutorials/how-to-install-SmartOS/how_to_install_SmartOS_without_the_need_for_USB_flash_drive.md]: license was added

**Comment by optaris** [comment]
Hello,

I made the requested changes, but I'm not sure what I have to do to have it merged.

Could you briefly indicate what's next?

kind regards
Stefan

File: tutorials/how-to-install-SmartOS/how_to_install_SmartOS_without_the_need_for_USB_flash_drive.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (rook59) [inline tutorials/how-to-install-SmartOS/how_to_install_SmartOS_without_the_need_for_USB_flash_drive.md:427]: Please sign off here

**Review by rook59 — APPROVED** [review]

**Pull request merged**